### PR TITLE
10mMOB -> 400uMOB fees

### PR DESCRIPTION
**Pull request by jcape**
Soundtrack of this PR: [link to song that really fits the mood of this PR](https://www.youtube.com/watch?v=OPkWYTUiPTo)

### Motivation

The fees are too damned high.

### In this PR
* Adjusts MINIMUM_FEE to 400uMOB.

### Future Work
* Configurable fees
* Consensus-controlled fees

### CODE REVIEW

File: transaction/core/src/constants.rs [hunk 1]
```diff
@@ -24,11 +24,14 @@ pub const MAX_TOMBSTONE_BLOCKS: u64 = 100;
...
 
+/// one microMOB = 1e6 picoMOB
+pub const MICROMOB_TO_PICOMOB: u64 = 1_000_000;
+
 /// one milliMOB = 1e9 picoMOB
 pub const MILLIMOB_TO_PICOMOB: u64 = 1_000_000_000;
 
 /// Minimum allowed fee, denominated in picoMOB.
-pub const MINIMUM_FEE: u64 = 10 * MILLIMOB_TO_PICOMOB;
+pub const MINIMUM_FEE: u64 = 400 * MICROMOB_TO_PICOMOB;
```
Comment (UkoeHB) [inline transaction/core/src/constants.rs:34]: worth leaving a comment noting fee change?
```
/// fee change: v0 [0.01 MOB] -> v1 [0.0004 MOB]
```

**Review by eranrund — APPROVED** [review]

File: transaction/core/src/constants.rs (hunk 1 above)
Comment (jcape) [reply transaction/core/src/constants.rs:34]: I dunno, we have git... :-)

Comment (UkoeHB) [reply transaction/core/src/constants.rs:34]: Guess my testimony as 'crypto historian' is - it's way easier to figure out what happened in the past if it's explicitly documented.

Comment (jcape) [reply transaction/core/src/constants.rs:34]: It is explicitly documented, in the commit messages.

**Review by mfaulk — APPROVED** [review]

**Pull request merged**